### PR TITLE
[Redesign] Text changes for all the Lorem ipsum.

### PR DESCRIPTION
**Pull request by jonwchu**

### CODE REVIEW

File: src/NuGetGallery/Views/Pages/Home.cshtml [hunk 1]
```diff
@@ -44,40 +44,36 @@
...
             <h1>What is NuGet?</h1>
             <p>
-                NuGet is the package manager for the Microsoft development platform including .NET. The NuGet client tools
-                provide the ability to produce and consume packages. The NuGet Gallery is the central package repository used
-                by all package authors and consumers.
+                NuGet is the package manager for .NET. The NuGet client tools provide the ability to produce and consume packages. 
+                The NuGet Gallery is the central package repository used by all package authors and consumers.
             </p>
-        </div>
+        </div>Lorem
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Pages/Home.cshtml]: `Lorem`?

File: src/NuGetGallery/Views/Shared/LayoutFooter.cshtml [hunk 2]
```diff
@@ -24,9 +24,9 @@
     <a title=".NET Foundation" href="https://www.dotnetfoundation.org"><img src="@Href("~/Content/Logos/dnf.png")" alt=".NET Foundation" /></a>
     <p>
         &copy; @DateTime.UtcNow.Year .NET Foundation -
+        <a href="@Url.Action("About", "Pages")">About</a> -
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Shared/LayoutFooter.cshtml]: Nit this is no longer used in the redesign -- but it makes sense for the admin pages

File: src/NuGetGallery/Views/Pages/About.cshtml [hunk 3]
```diff
@@ -13,9 +13,7 @@
         <div class="col">
             <h1>About NuGet</h1>
             <p>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-
-                Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                An essential tool for any modern development platform is a mecahnism through which developers can create, share, and consume useful libraries and packages. For .NET, that mechanism is NuGet, which defines how packages for .NET are created, hosted, and consumed, and provides the tools for each of those roles.
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Pages/About.cshtml]: mechanism

File: src/NuGetGallery/Views/Pages/Home.cshtml [hunk 4]
```diff
@@ -44,40 +44,36 @@
...
             <div class="col-sm-4">
                 <img width="225" height="225" alt="Cartoon image of a person lifting a package into the clouds"
                      src="~/Content/gallery/img/orange-circle.svg"
                      @ViewHelpers.ImageFallback(Url.Absolute("~/Content/gallery/img/orange-circle-225x225.png")) />
-                <h3>Upload</h3>
+                <h3>Learn</h3>
                 <p>
-                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed porttitor, neque id consectetur fringilla, odio
-                    justo consectetur leo, nec interdum odio odio non lacus. Donec commodo enim et suscipit ultrices.
+                    Include New to NuGet? Start with a walkthrough showing how NuGet powers your .NET development.
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Pages/Home.cshtml]: Should be just "New to NuGet"

File: src/NuGetGallery/Views/Pages/Home.cshtml [hunk 5]
```diff
@@ -44,40 +44,36 @@
...
-                     @ViewHelpers.ImageFallback(Url.Absolute("~/Content/gallery/img/blue-circle-225x225.png")) />
-                <h3>Download</h3>
+                <img width="225" height="225" alt="Cartoon image of a person looking into the clouds with a telescope"
+                     src="~/Content/gallery/img/purple-circle.svg"
+                     @ViewHelpers.ImageFallback(Url.Absolute("~/Content/gallery/img/purple-circle-225x225.png")) />
+                <h3>Explore</h3>
                 <p>
-                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed porttitor, neque id consectetur fringilla, odio
-                    justo consectetur leo, nec interdum odio odio non lacus. Donec commodo enim et suscipit ultrices.
+                    Browse the 80,000+ packages that developers like you have created and share with the .NET community.
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Pages/Home.cshtml]: Are we okay with a number in static texts like this? We'll have to keep updating it periodically, which will be a bit of a nuisance.

Comment (loic-sharma) [inline src/NuGetGallery/Views/Pages/Home.cshtml]: Also "share" should be "shared"

File: src/NuGetGallery/Views/Pages/About.cshtml (hunk 3 above)
Comment (loic-sharma) [inline src/NuGetGallery/Views/Pages/About.cshtml]: The second sentence feels very repetitive in this section. How about something like this?

> An essential tool for any modern development platform is a mechanism through which developers can create, share, and consume useful libraries and packages. NuGet is that tool for .NET.

**Pull request merged**